index.js

```diff
@@ -5,6 +5,12 @@ var readline = require('readline');
 const Gamedig = require('gamedig');
 const {searchResultsWhereNameAndType, getInfoFromName} = require('myanimelists');
 
+//command requires

```

Comment:
Hmmm. I thought the plan was to start using the import approach, instead of require?